hypha/apply/funds/tables.py

```diff
@@ -188,8 +191,16 @@ class Meta(SubmissionsTable.Meta):
             "screening_status",
             "reviews_stats",
             "organization_name",
-        )
-        sequence = fields + ("comments",)
+        ]
+
+        # Remove fields that have been added to the exclude fields settings.
+        fields = [
+            x for x in fields if x not in settings.SUBMISSIONS_TABLE_EXCLUDED_FIELDS
+        ]
```

Comment:
It doesn't seem to be working that way. I looked and tried to update its base columns and sequence in `__init__` then it works.
here is the code:
```    
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)

        excluded_columns = getattr(settings, "SUBMISSIONS_TABLE_EXCLUDED_FIELDS", [])
        for column_name in excluded_columns:
            if column_name in self.base_columns:
                del self.base_columns[column_name]
        self.sequence = [column_name for column_name in self._meta.fields if column_name not in excluded_columns]```

Reply:
I've added the code.